tests/frameworks/gtest/include/gtest-api.h

```diff
@@ -97,13 +97,9 @@ std::vector<char> Format(NSString* message, Args ... args)
 }
 }
 
-namespace testing {
-namespace internal {
```

Comment:
can we instead change the macro definitions to use the namespace? this change adds a global operator overload in a .h which includers may not want ...